Source/FakeItEasy.Specs/GlobalSuppressions.cs

```diff
@@ -6,6 +6,22 @@
     Target = "FakeItEasy.Specs.when_faking_a_disposable_class.#.ctor()",
     Justification = "Required for testing.")]
 
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(
+    "Microsoft.Design",
+    "CA1045:DoNotPassTypesByReference",
+    MessageId = "0#",
+    Scope = "member",
+    Target = "FakeItEasy.Specs.when_matching_a_call_with_a_ref_parameter+IHaveInterestingParameters.#CheckYourReferences(System.String&)",
+    Justification = "Required for testing.")]
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage(
+    "Microsoft.Design",
+    "CA1021:AvoidOutParameters",
+    MessageId = "0#",
+    Scope = "member",
+    Target = "FakeItEasy.Specs.when_matching_a_call_with_a_parameter_having_an_out_attribute+IHaveInterestingParameters.#Validate(System.String)",
+    Justification = "Required for testing.")]

```

Comment:
Did VS give you the option to add these suppressions in the code file itself? Generally I prefer that over `GlobalSuppressions.cs`.